.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@
 exclude: migrations
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.1.0
+    rev: v5.0.0
```

Comment:
I upgraded this repo whilst I was changing things because the old version hit a deprecation warning from pre-commit:

```
[WARNING] repo `https://github.com/pre-commit/pre-commit-hooks` uses deprecated stage names (commit, push) which will be removed in a future version.  Hint: often `pre-commit autoupdate --repo https://github.com/pre-commit/pre-commit-hooks` will fix this.  if it does not -- consider reporting an issue to that repo.
```

There are no major changes